src/main/java/Pixel.java

```diff
@@ -0,0 +1,44 @@
+import java.awt.*;
+import java.awt.image.BufferedImage;
+
+public class Pixel {
+    private int energy;
+    private Color brightness;
+
+    public Pixel() {
+    }
+
+    public void setEnergy(int borderEnergy) {
+        energy = borderEnergy;
+    }
+
+    public int getEnergy() {
+        return energy;
+    }
+
+    public void setEnergy(int abovePixel, int belowPixel, int leftPixel, int rightPixel) {
+        Color abovePixelColor = new Color(abovePixel);
+        Color belowPixelColor = new Color(belowPixel);
+        Color rightPixelColor = new Color(rightPixel);
+        Color leftPixelColor = new Color(leftPixel);
```

Comment:
Try saving the values from the Image as Color objects, rather than recreating them each time.